server/plugin/api.go

```diff
@@ -58,6 +58,17 @@ const (
 	ResponseTypeJSON ResponseType = "JSON_RESPONSE"
 	// ResponseTypePlain indicates that response type is text plain
 	ResponseTypePlain ResponseType = "TEXT_RESPONSE"
+	RepoName          string       = "repo_name"
+	RepoOwner         string       = "repo_owner"
+	IssueNumber       string       = "issue_number"
+	IssueID           string       = "issue_id"
+	Status            string       = "status"
+	ChannelID         string       = "channel_id"
+	PostID            string       = "postId"
+
+	CommentModalHeader string = "comment_modal"
```

Comment:
```suggestion
	HeaderCommentModal string = "comment_modal"
```
Common part should be used as prefix in constant names

Reply:
Also it should be `WebsocketEventOpenCommantModal` and the value should be `open_comment_modal`